projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -308,11 +343,11 @@ export class IgxInputGroupComponent extends DisplayDensityBase
      */
     @HostBinding('class.igx-input-group--border')
     public get isTypeBorder() {
-        return this._type === 'border';
+        return this._type === 'border' && this._variant === 'material';
     }
 
     /**
-     * Returns whether the `IgxInputGroupComponent` type is Fluent.
+     * Returns true if the `IgxInputGroupComponent` theme is Fluent.
```

Comment:
woof, no idea why those (is fluent, is indigo, has border, etc) are actually public. Seems a lot like those should be `@hidden @internal` TBH

Reply:
I don't like it either, but I also don't think it should be handled in this PR.